src/isomorphic/React.js

```diff
@@ -28,6 +28,13 @@ var createFactory = ReactElement.createFactory;
 var cloneElement = ReactElement.cloneElement;
 
 if (__DEV__) {
+  console.log(
```

Comment:
@aweary my understanding is that deduping is happening there because it's a function that can be called many times. Isn't this file meant to be evaluated once?